src/main/java/sparksoniq/exceptions/codes/ErrorCodes.java

```diff
@@ -50,6 +50,8 @@
     public static final String UnimplementedErrorCode = "RBST0002";
     public static final String JobWithinAJobErrorCode = "RBST0003";
     public static final String OurBadErrorCode = "RBST0004";
+    public static final String UnknownRumbleMLClassReferenceErrorCode = "RBST0005";
+    public static final String UnknownRumbleMLParamReferenceErrorCode = "RBST0006";
 
```

Comment:
@ghislainfourny Do you have a preference for Error codes & naming? 

TODO: Update error code documentation once this is settled.

Reply:
Discussed offline, all clear